gprofiler/utils/__init__.py

```diff
@@ -206,8 +208,8 @@ def run_process(
     kill_signal: signal.Signals = signal.SIGKILL,
     communicate: bool = True,
     stdin: bytes = None,
-    **kwargs,
-) -> CompletedProcess:
+    **kwargs: Any,
+) -> "CompletedProcess[bytes]":
```

Comment:
```suggestion
) -> CompletedProcess[bytes]:
```

(it's already declared)

Reply:
`CompletedProcess` isn't generic at runtime, only at type check time (it's generic in `typeshed`), so removing the double quotes results in `TypeError: 'type' object is not subscriptable`.

See [this](https://stackoverflow.com/questions/68089860/completedprocess-is-a-generic-for-mypy)

If we were to drop support for Python 3.6, we could remove the double quotes, and `from __future__ import annotations`

Reply:
Blehhh

We will drop 3.6 support at some point, but not in this PR :sweat_smile: